src/server/run-server.ts

```diff
@@ -40,5 +40,27 @@ export function runServer(db: Knex, augur: Augur): RunServerResult {
     }
   });
 
+  app.get("/status/database", (req, res) => {
+    const maxPendingTransactions: number = (typeof req.query.max_pending_transactions === "undefined") ? 1 : parseInt(req.query.max_pending_transactions, 10);
+    if (isNaN(maxPendingTransactions)) {
+      res.status(400).send({error: "Bad value for max_pending_transactions, must be an integer in base 10"});
```

Comment:
And to respond to your previous review, this should technically probably by 422 (Unprocessible Entity), since its an error with Input but the content types are alright.

Reply:
Yea i was wrong on both comments :)